starport-core/src/main/scala/com/krux/starport/util/ErrorHandler.scala

```diff
@@ -81,9 +80,11 @@ object ErrorHandler extends Logging with WaitForIt {
   /**
    * @return the SES send ID
    */
-  def cleanupActivityFailed(pipeline: Pipeline, stackTrace: Array[StackTraceElement]): String = {
+  def cleanupActivityFailed(pipeline: Pipeline, stackTrace: Array[StackTraceElement])
+    (implicit conf: StarportSettings): String = {
     SendEmail(
-      Seq(PagerDutyEmail, "kexin.xie@salesforce.com"),
+      pipeline.owner.map(Seq(_)).getOrElse(conf.toEmails),
```

Comment:
Cleanup actually does not need to notify the pipeline owners, `toEmails` here is fine, since typically no action is required, mainly to let service owners know. Maybe we should add a comment here.

Reply:
done.